### PR TITLE
React-Capstone : Coin tracker

### PR DESCRIPTION
**Pull request by ShaliniNayan**
Hello,

For this PR, I have a Capstone Project and my project name is Coin Tracker initialize the project using:

Setup new React application using Create React App
Loom video link : https://www.loom.com/share/0aae677513954484ba479aed88a1fb37?sid=6427f8fa-3c54-4f07-b9a9-802ccdceb66b 

Interactions:

- Home page:
- When the page loads, the web app shows the list of categories that could be filtered out by some parameter.
- Along with the category name, it will display some numeric values per category that come from the API.
- When the user clicks (or taps) on a category item, the application navigates to the details page.

- Details page:
     - In the details page, the web app retrieves data from the API to show detailed data in the category.
     - When the user clicks on the "Back" button, the user navigates to the home page.
     
- Testing requirements
     - Create unit tests for pure functions in your Redux code.
     - Create integration tests for your application using the React Testing Library.
     - Mock the access to the API so that tests don't send actual requests.
     - Mock the connection to the Redux Store.
  
- Gitflow follows.

- No linter error.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,8 +63,8 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 > Below are important links to live project .
-- [Live Demo Link]()
-- [Loom Video Recording]()
+- [Live Demo Link](https://cointracker-smct.onrender.com/)
```
Comment (rustam-tolipov) [inline README.md:66]: **Well done!
You did a great job!**

- [x] This is the only issue that I was able to find. In the testing part of your app, I can see only the components test. But, in the mock testing folder, I was not able to find any tests. Please add mock tests to your project and make sure they are passing as expected!

HINT: You can perform a mocking data fetching and Redux Store connection.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @ShaliniNayan,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Well-implemented code_
_Kudos for adding a search bar_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ShaliniNayan) [reply README.md:66]: Hello,
- Make the correction in the README.md file.
- Create coinsContaine.test.js file in the __test__ folder and it's passed successfully.

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @ShaliniNayan ,
# STATUS: APPROVED! 🥇 💯 
done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**